backend/src/models/user.js

```diff
@@ -17,7 +18,25 @@ const UserSchema  = new mongoose.Schema({
         type: String,
         required: true,
         unique: true
-    }
+    },
+    firstName: {
+        type: String,
+        required: true
+    },
+    surname: {
+        type: String,
+        required: true
+    },
+    phone: String,
+    picture: {
+        large: {data: Buffer, imageType: String},
+        thumbnail: {data: Buffer, imageType: String}
+    },
+    certificate: {
+        data: Buffer, imageType: String
+    },
+    isInstructor: Boolean,
+    tours: [{ type: ObjectId, ref: 'Tour' }]
```

Comment:
Same as isInstructor, we already have an relation in the tours schema, please check if it's ok, maybe change it to this better way and make the relation both-way.